### ISSUE TITLE
question about the inclusion of python

### ISSUE THREAD

**Issue by dkarter** [adde88/sslstrip-hsts-openwrt#1]
Thank you for writing up this package for the pineapple! ❤️

I was wondering why the install script will attempt to delete python from the pineapple and why you have decided to include your own version of python with the package as opposed to installing it from the original vendor?

**Comment by adde88** [comment]
The install-script cleans the python-dir, and then uses this version, which is the same version shipped with the Pineapples + the nessecary libraries needed to run SSLstrip+ and dns2proxy.

The reason i had to do this, instead of creating packages for all the python modules that are needed was because i had alot of issues compiling them with the openwrt-sdk.
I managed to compile them in the end with a lot of tweaking, but the SDK wouldn't let me create individual installation packages for those modules.
I then took the entire python-lib from the SDK and copied it manually over to my Pineapple for testing.
And it worked.

I then manually created a IPK/opkg-package containing this working python-lib.
It deletes the directory to ensure that the python-dir will be working.
On the Pineapple Nano it also deletes the one on the internal storage to rather use the SD-card, and then create a sym-link to the internal storage. (Saved a lot of space)

I know this is a dirty method to do it. And it requires a bit trust in me, as you don't know what this folder contains.
But it was the only thing i managed to get working at that moment.
Though i will continuously be trying to get the modules compiled properly.
